source/admin_manual/list_of_events.rst

```diff
@@ -1242,13 +1242,20 @@ Finished the message submission.
 Push notifications
 ==================
 
-+-----------------------+------------------------------------------------------+
-| Field                 | Description                                          |
-+=======================+======================================================+
-| Inherits from mail user                                                      |
-+-----------------------+------------------------------------------------------+
-| *No event specific fields defined*                                           |
-+---------------------+--------------------------------------------------------+
+.. versionadded:: 2.3.10
+
+   ``mailbox``, ``mail_uid``, ``failed`` and ``push_notification_event`` were added.
+
++-------------------------+------------------------------------------------------+
+| Field                   | Description                                          |
++=========================+======================================================+
+| Inherits from mail user                                                        |
++-------------------------+------------------------------------------------------+
+| mailbox                 | Mailbox for event                                    |
+| mail_uid                | Mail uid (when present)                              |
+| failed                  | Set to ``yes`` on failure.                           |
```

Comment:
Hrm.  Should we use booleans instead?  I suppose the event code doesn't support them at the moment.

Reply:
well, it's kinda of boolean. it's only there when it fails, so you can match it with `failed=*`

Reply:
Sure. I was thinking about explicit bool values - e.g., true/false in json.  Anyway, that's for another day.